### ISSUE TITLE
Debug Console, Copy All (to clipboard) includes source file name

### ISSUE THREAD

**Issue by timmackey** [microsoft/vscode#108149]
- VSCode Version: 1.49.3
- OS Version: Windows 10

Steps to Reproduce:

1. insert "console.log('anything');" in code that will be executed many times.
2. view Debug Console window
3. right click, select "Copy All"
4. open some other text editor app (notepad, Notepad++)
5. paste clipboard
6. observe new line identifying source module containing 'console.log' statement.
i.e. "s request_ExamFee mS = 673
 src/app/ttm/ttm-console-log.component.ts"
7. VS Code version 1.48.2 does not include file name.
i.e.  "s request_ExamFee mS = 673"

For many years I have been relying on "Copy All" to **not** include the source file name for diagnostics.  In my case all 'console.log' statements are in a file specifically constructed to enable/disable debugging categories on a global basis, hence the source file name is redundant and irrelevant.
In my case "src/app/ttm/ttm-console-log.component.ts" on a new line is redundant text repeated for every 'console.log'.
Just image thousands of extra lines in your text editor containing the same unwanted text interspersed with helpful diagnostic data. 
In the rare situations were I want to know the source file, I go to the browser console and copy the console log from there.
(I think I did that maybe once in the past 3 years.)

I have not been able to find a bug report that this was an issue, nor any notes that this "fix" was implemented.
(They may exist.  I just haven't been able to find it.  And I haven't searched the code base.)

If VS Code *must* fix something that I considered a feature, not a bug, please make this "fix" configurable so that it can be disabled.  As long as Debug Console includes the source file name in a "Copy All", I will be stuck at version1.48.2.

Does this issue occur when all extensions are disabled?: Unable to test without "Debugger for Chrome" extension installed.
VS Code error dialog reports: "Configured debug type 'chrome' is not supported." when attempting to start VS Code debugger.
Launching with '--disable-extensions' produces the following warnings:

`C:\Program Files\Microsoft VS Code>code --disable-extensions

C:\Program Files\Microsoft VS Code>
[main 2020-10-06T00:48:24.811Z] update#setState idle
(node:20028) Electron: Loading non-context-aware native module in renderer: '\\?\C:\Program Files\Microsoft VS Code\resources\app\node_modules.asar.unpacked\vscode-sqlite3\build\Release\sqlite.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:20028) Electron: Loading non-context-aware native module in renderer: '\\?\C:\Program Files\Microsoft VS Code\resources\app\node_modules.asar.unpacked\node-pty\build\Release\conpty.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:20028) Electron: Loading non-context-aware native module in renderer: '\\?\C:\Program Files\Microsoft VS Code\resources\app\node_modules.asar.unpacked\windows-process-tree\build\Release\windows_process_tree.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:20028) Electron: Loading non-context-aware native module in renderer: '\\?\C:\Program Files\Microsoft VS Code\resources\app\node_modules.asar.unpacked\spdlog\build\Release\spdlog.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
(node:19280) Electron: Loading non-context-aware native module in renderer: '\\?\C:\Program Files\Microsoft VS Code\resources\app\node_modules.asar.unpacked\spdlog\build\Release\spdlog.node'. This is deprecated, see https://github.com/electron/electron/issues/18397.
[main 2020-10-06T00:48:54.813Z] update#setState checking for updates
[main 2020-10-06T00:48:55.173Z] update#setState idle`

**Comment by isidorn** [comment]
Next time please concise issue description, this reads more like a novel and I already read War and Peace.

We have included the source file name due to a different request.
Though it might make sense for Copy All to not contain the source file name.

Anyways leaving this open for now so we gather more user feedback and we are open to changing this in the future.
Thanks!

**Comment by timmackey** [comment]
@isidorn - The issue template invited a concise description.  A concise description has been provided.
The issue template requests steps to reproduce.  They are provided.
Also provided is the rational for my request, and an example of the issue.
If the team wants less information in an issue report, I suggest asking for less information in the template.

I, and I believe others in this community, would appreciate contributors addressing issues without sarcastic commentary.  I doubt you have actually read War and Peace.
If there are too many words in this report, I can learn to submit multiple reports (referencing each other) with fewer words, if that's what the team really wants.

I appreciate the issue has been added to Backlog and under-discussion.
Not wanting to repeat my entire rationale here, please read it in the report.
To summarize: please don't "fix" things that aren't broken (and then not report the "fix" in the release notes).
Thank you.

**Comment by isidorn** [comment]
I have actually read War and Peace. I can highly recommend it.

**Comment by timmackey** [comment]
@isidorn - "We have included the source file name due to a different request."
Could you please identify the request # so that I may be better able to understand your decision?
Thank you.

**Comment by isidorn** [comment]
This was the motivation https://github.com/microsoft/vscode/issues/105868
I was wrong, nobody requested that copy all should include the source file name. It is just that the filtering and copyAll use the same `toString()` implementation. So I can reopen this for now

**Comment by timmackey** [comment]
Could you please identify the change set that caused inclusion of the source file name?
Thank you.

**Comment by isidorn** [comment]
https://github.com/microsoft/vscode/commit/5dc43b2bd39f27d8861e47b1169887376e6df26c

**Comment by timmackey** [comment]
@isidorn - Please.  Restoring this to the original "not broken" implementation would seem to entail a trivial amount of effort.
Was anyone complaining about the filtering to cause the team to consider implementing the current code in the first place?
If yes, please specify issue #.  If no, why can't this be rolled back to version 1.48.2 implementation?

**Comment by timmackey** [comment]
@isidorn
May I suggest adding a toString() function specifically for CopyAll()?  i.e. toStringSansSource()

**Comment by timmackey** [comment]
Gentlemen - I appreciate your consideration to implement my request.
Examining the implementation, I would have expected  `e.toString(true)` in debug/browser/repl.ts to be `e.toString(false)` to inhibit inclusion of the source file name.

**Comment by isidorn** [comment]
I think we are good. Try it out with vscode insiders from Friday and if you find some corner case where it does not work let me know. Thanks

**Comment by timmackey** [comment]
@isidorn - Visual Studio Code Insiders published Monday 2/8/2021 contains the requested changes.
Thank you.

**Comment by isidorn** [comment]
Based on your comment adding verified label.
